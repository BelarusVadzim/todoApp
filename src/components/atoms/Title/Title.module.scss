@import 'src/variables.scss';

.title {
  border: none;
  color: white;
  cursor: default;
  font-family: Arial, Helvetica, sans-serif;
  font-size: 2.5rem;
  font-weight: bold;
  letter-spacing: 10px;
}

