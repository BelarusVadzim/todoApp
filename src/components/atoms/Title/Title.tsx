import React, { PropsWithChildren } from 'react';
import style from './Title.module.scss';

const Title: React.FC<PropsWithChildren> = ({ children }) => {
  return <span className={style.title}>{children}</span>;
};

export default Title;
