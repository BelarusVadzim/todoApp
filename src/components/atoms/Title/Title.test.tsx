import { render } from '@testing-library/react';
import Title from './Title';

jest.mock('./Title.module.scss', () => ({ title: 'titleStyle' }));

describe('<Title />', () => {
  it('should render properly', () => {
    const component = render(<Title>Title text</Title>);

    expect(component.asFragment()).toMatchSnapshot();
  });
});
