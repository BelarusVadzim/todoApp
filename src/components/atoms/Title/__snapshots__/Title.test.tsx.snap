// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render properly 1`] = `
<DocumentFragment>
  <span
    class="titleStyle"
  >
    Title text
  </span>
</DocumentFragment>
`;
